pkgs/games/kobodeluxe/default.nix

```diff
@@ -9,23 +9,33 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ SDL SDL_image libGLU libGL ];
+  nativeBuildInputs = [ makeWrapper ];
 
   prePatch = ''
     sed -e 's/char \*tok/const char \*tok/' -i graphics/window.cpp
   '';
 
+  # We wrap the program to give it the highscore directory, otherwise the scores can't be saved
+  # because the nix store directory where the game lives is read only
+  postInstall = ''
+    wrapProgram $out/bin/${meta.mainProgram} \
+      --run "mkdir -p ~/.config/${pname}" \
+      --add-flags "-scores ~/.config/${pname}"
+  '';
+
   patches = [
-    (fetchpatch {
+    (fetchpatch { # Fixes a build failure: error const enemy kind pipe2 redeclared as different kind of symbol
       url = "https://sources.debian.org/data/main/k/${pname}/${version}-10/debian/patches/04_enemies-pipe-decl.patch";
       sha256 = "sha256-myMhB7+7y6EeD+xIfeHTbJPLrulyQmzpTxuNX1azpSY=";
     })
-    (fetchpatch {
+    (fetchpatch { # Fixes the game being stuck in the pause state
```

Comment:
Nit: Could you add these in the commit that adds the patches?

Reply:
done